primitives/core/src/offchain/error.rs

```diff
@@ -0,0 +1,18 @@
+#![allow(missing_docs)]
```

Comment:
Missing pre amble. `missing_docs` should be defined on the crate level, isnt' it?

Reply:
I will check but I think it is on warn at crate level and I needed this file to be on allow

Reply:
I will add doc

Reply:
done